libtsuba/include/tsuba/RDG.h

```diff
@@ -228,7 +246,10 @@ class KATANA_EXPORT RDG {
 
   void InitEmptyTables();
 
-  katana::Result<void> DoMake(const katana::Uri& metadata_dir);
+  katana::Result<void> DoMake(
+      const std::vector<PropStorageInfo*>& node_props,
+      const std::vector<PropStorageInfo*>& edge_props,
+      const katana::Uri& metadata_dir);
```

Comment:
Is it true that when this is called with a subset of node/edge properties, only that subset is loaded, but the entire set of properties is retained by the RDG?

Reply:
yes.

Reply:
Then you might want to call these `loaded_node_props` and `loaded_edge_props`.